### PR TITLE
Ports sec hailer dispatch system from NSV13

### PR DESCRIPTION
**Pull request by maxion12345**
# General Documentation

### Intent of your Pull Request

This adds a neat "dispatch" button to security hailers used on security masks / swat masks / cyborg hailers. This lets you quickly report crimes and a location with the click of a button to rapidly get backup. Mainly for immersion, it's just pretty neat. Currently only can dispatch for certain crimes. 

### Why is this change good for the game?

Muh immersion. Also, sechailers aren't used much outside of "wirecut the restrictor, spam it to annoy people" so this will give them added functionality.
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
dont think we even have a wiki page for sechailers

### What should players be aware of when it comes to the changes your PR is implementing?
Sec hailers can now report location and crimes to other officers. 
### What general grouping does this PR fall under? 
Security tweak

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

# Changelog

:cl:  
rscadd: Added sechailer dispatch system
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
Wait it compiled? What the hell I tried that earlier

I'll futz with it later, thanks for the help

File: yogstation/code/modules/clothing/masks/hailer.dm [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	radio.listening = FALSE
+	radio.recalculateChannels()
+
+/obj/item/clothing/mask/gas/sechailer/proc/dispatch(mob/user)
+	var/area/A = get_area(src)
+	if(world.time < last_dispatch + dispatch_cooldown)
+		to_chat(user, "<span class='notice'>Dispatch radio broadcasting systems are recharging.</span>")
+		return FALSE
+	var/list/options = list()
+	for(var/option in list("401 (murder)", "101 (resisting arrest)", "210 (breaking and entering)", "206 (riot)", "302 (assault on an officer)")) //Just hardcoded for now!
```
Comment (athena413) [inline yogstation/code/modules/clothing/masks/hailer.dm]: In our Space law it is:
601 (murder)
101 (res. arrest)
310 OR 410 (B&E or B&E Restricted)
306 (Rioting) OR 406 (Inc. Riot)
401 (Assault, Officer)

**Review by athena413 — CHANGES_REQUESTED** [review]
inconsistent space law crime codes

**Comment by maxion12345** [comment]
alright it complies in the checks, im guessing whatever the error is on my side, im pulling a cyberpunk and releasing this pr anyways

**Review by nesisknesis — APPROVED** [review]

**Comment by 00aquir** [comment]
Could you make the HUD icon just one image? It is kind of distracting when it keeps changing.
